site/src/docs/components/header/code.mdx

```diff
@@ -304,7 +304,95 @@ import { Header, IconSearch } from 'hds-react';
 
 </Playground>
 
-**Note:** Header is designed to be used full width at the top of the page above the main content. Preview here might not scale correctly, <Link openInNewTab openInNewTabAriaLabel="Opens in a new tab" href="/storybook/react/?path=/story/components-header--with-full-features-dark-theme">view full header with dark theme example in Storybook</Link>.
+#### Custom theme
+
+<Playground scope={{ Header, IconSearch }}>
+
+```jsx
+import { Header, IconSearch } from 'hds-react';
+
+() => (
+  <Header theme={{
+    '--color-focus-outline': 'var(--color-metro-dark)',
+    '--header-color': 'var(--color-black)',
+    '--actionbar-background-color': 'var(--color-engel)',
+    '--nav-background-color': 'var(--color-engel-light)',
+    '--nav-border-color': 'var(--color-black)',
+    '--nav-link-hover-color': 'var(--color-black)',
+    '--universal-bar-background-color': 'var(--color-black-20)',
+    '--nav-link-dropdown-background-color': 'var(--color-white)',
+    '--nav-button-background-color': 'var(--color-black-20)',
+    '--nav-button-hover-background-color': 'var(--color-black-40)',
+    '--nav-drop-down-icon-color': 'var(--color-black)',
+    '--header-background-color': 'transparent',
+  }}>
+    <Header.SkipLink skipTo="#content" label="Skip To Content" />
+    <Header.UniversalBar primaryLinkHref="#" primaryLinkText="Helsingin kaupunki">
+      <Header.Link href="#" label="Uutiset" />
+      <Header.Link href="#" label="Asioi verkossa" />
+      <Header.Link href="#" label="Anna palautetta" />
+    </Header.UniversalBar>
+    <Header.ActionBar
+      title="Helsingin kaupunki"
+      titleAriaLabel="Helsingin kaupunki"
+      titleHref="https://hel.fi"
+      logoAriaLabel="Service logo"
+      logoHref="https://hel.fi"
+      menuButtonAriaLabel="Menu"
+    
```

Comment:
Missing the logoSrc right about here? 😁 Dark theme code Playground is missing it as well.

Reply:
Ah yes, only empty line... well not logoSrc but the <Logo /> :D

Reply:
fixed